### DISCUSSION TITLE
General question: Creative Tech companies/marketing companies that focus on non-profits/public affairs/civics/advocacy?

### DISCUSSION THREAD

**Discussion by EdwardDeaver** [j0hnm4r5/awesome-creative-technology#80] [General]
Hi John, 

I'm not sure what to Google to find these companies but are there installation/creative tech marketing companies that focus on non-profit clients or advocacy or civics?

I found Media Cause https://mediacause.com/ but that's it.

Do you know of others?

-Edward

**Comment by j0hnm4r5** [comment]
Hmmm that's a good question. I can't think of any off the top of my head. I wish we had a better way to really query all of these that's more than just Cmd+f with the keywords.

Maybe a custom GPT or something?
